app/cafe.py

```diff
@@ -0,0 +1,28 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:

```

Comment:
`if visitor.get("vaccine")` is more efficient since you don't need to iterate through all the keys in visitor dict but you access element if it exists instantly